### ISSUE TITLE
[BUG] "Move" button not displayed for some tokens

### ISSUE THREAD

**Issue by jeden** [onflow/FRW-iOS#602]
The `Top Shot $BETA` token should have the "Move" button enabled, but it's hidden instead.

This is a dump of the token

```
▿ TokenModel
  - name : "Top Shot $BETA"
  ▿ address : FlowNetworkModel
    ▿ mainnet : Optional<String>
      - some : "0xee09029f1dbcd9d1"
    - testnet : nil
    - crescendo : nil
    - previewnet : nil
  - contractName : "TopShotBETA"
  ▿ storagePath : FlowTokenStoragePath
    - balance : "/public/TopShotBETAMetadata"
    - vault : "/storage/TopShotBETAVault"
    - receiver : "/public/TopShotBETAReceiver"
  - decimal : 8
  ▿ icon : Optional<URL>
    ▿ some : https://nftstorage.link/ipfs/bafkreifkjlkzfjml77ogwnxnbqbeixbxau6lyubwtrcfsi3uwaeep664mu
      - _url : https://nftstorage.link/ipfs/bafkreifkjlkzfjml77ogwnxnbqbeixbxau6lyubwtrcfsi3uwaeep664mu
      - _parseInfo : nil
      - _baseParseInfo : nil
  ▿ symbol : Optional<String>
    - some : "BETA"
  - website : nil
  - evmAddress : nil
  - flowIdentifier : nil
  - balance : nil
```

The BETA token has EVM address on our token list
https://github.com/Outblock/token-list-jsons/blob/outblock/jsons/mainnet/flow/default.json#L785-L807

But it's `nil` in the dump.

Figure out why there is no EVM address.

**Comment by jeden** [comment]
Fixed by Caos. Updated the`https://raw.githubusercontent.com/Outblock/token-list-jsons/outblock/jsons/mainnet/flow/dev.json` file
[Link to commit](https://github.com/Outblock/token-list-jsons/commit/7cef424c14e6bfa971cffb446a7e527ac0983fee)

**Comment by Peppermint1020** [comment]
This issue fixed in ios dev version 2.2.10(34)
![Image](https://github.com/user-attachments/assets/ad983515-fa23-4a4a-984e-94f0d0bcd605)
